modules/understanding-upgrade-channels.adoc

```diff
@@ -99,6 +99,32 @@ endif::openshift-origin[]
 
```

Comment:
do we want to fiddle with "after the update recommendation has been removed" a few lines up^^?  Maybe "after the update recommendation has been removed or been made conditional" or some such?

Reply:
My only hesitation with this was that we were going to drop new content in the middle of a section. So, I figured at least its own section would be better and would be easier for linking and standing out.

Reply:
I like having the new section.  I just think that "after the update recommendation has been removed" is a bit stale in this new context, and that we probably want to expand it so that folks know when they're supported.  Cases:

* Update is currently recommended for your cluster: definitely supported.
* Update was previously recommended, but is now completely gone from the recommended-update chart: still supported, but unlikely to happen anymore with 4.10 -> * updates, because we'll use a conditional edge.
* Update was previously recommended, but is now conditionally recommended, and for the cluster you're looking at, a risk applies, and it's not recommended: still supported.  This is the bit I think we want to call out new in this PR.
* Update was never recommended in any supported channel: not supported.

Reply:
> ifdef::openshift-origin[]
The presence of an update recommendation in the `stable-4` channel at any point is a declaration that the update is supported. While releases will never be removed from the channel, update recommendations that exhibit serious issues will be removed from the channel. Updates initiated after the update recommendation has been removed are still supported.
endif::openshift-origin[]

Update to:

ifdef::openshift-origin[]
The presence of an update recommendation in the `stable-4` channel at any point is a declaration that the update is supported. While releases will never be removed from the channel, update recommendations that exhibit serious issues will be removed from the channel. Update that was previously recommended, but is now conditionally recommended, and for the cluster, you wish to update, a risk still applies, and it is not recommended for an update is supported. 
endif::openshift-origin[]

Reply:
> ...update recommendations that exhibit serious issues will be removed from the channel...

Ah, this is going to be stale too.  Now it will be "may be removed from the channel, or may become conditional updates", or some such.

> Update that was previously...

"An update that..."?  Also, the rest of this sentence has a lot of conditional comments.  The main point is that if the update was ever recommended in a supported channel for any cluster will always be supported for every cluster.  What if they become conditional?  Still supported.  What if they are removed?  Still supported.  Maybe something like:

    Once an update recommendation is supported, it remains supported for the life of {product-version}, even if the update recommendation is later dropped or made conditional.

Also, looks like there is similar wording for OCP and OKD, and we should probably update both, although I'm a bit fuzzy on what "supported" means for OKD.

```console
$ git grep -n 'The presence of an update recommendation' | cut -b -180
modules/understanding-upgrade-channels.adoc:91:Update stability depends on your channel. The presence of an update recommendation in the `candidate-{product-version}` channel does 
modules/understanding-upgrade-channels.adoc:97:The presence of an update recommendation in the `stable-4` channel at any point is a declaration that the update is supported. While 
```

Reply:
Done